src/components/NavButton.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+function NavButton({buttonContent, disabled}) {
+    return (
+        <button disabled={disabled}>{buttonContent}</button>
```

Comment:
Hier mist nog de onClick attribute waarbij de name in de console moet verschijnen.
`onClick={() => ... }`